flutter/shell/platform/tizen/flutter_tizen_engine.cc

```diff
@@ -177,6 +177,14 @@ bool FlutterTizenEngine::RunEngine() {
     custom_task_runners.render_task_runner = &render_task_runner;
   }
 
+  if (project_->ui_thread_policy() !=
+      FlutterUIThreadPolicy::kRunOnSeparateThread) {
+    FT_LOG(Info) << "Running with merged platform and UI threads.";
+    custom_task_runners.ui_task_runner = &platform_task_runner;
+  } else {
+    FT_LOG(Info) << "Running with unmerged platform and UI threads.";
```

Comment:
I don't think it's necessary to always output the info log. If default value is explicitly stated in the documentation, I believe it's sufficient to output logs only when something is configured.